R/edit.R

```diff
@@ -63,13 +67,27 @@ edit_git_ignore <- function(scope = c("user", "project")) {
 #' @param type Snippet type. One of "R", "markdown", "C_Cpp", "Tex",
 #'   "Javascript", "HTML", "SQL"
 edit_rstudio_snippets <- function(type = "R") {
+  create_directory("~", ".R")
+  create_directory("~", ".R/snippets")
   edit_file("~", paste0(".R/snippets/", tolower(type), ".snippets"))
   invisible()
 }
 
-scope_dir <- function(scope = c("user", "project")) {
+scope_dir <- function(scope = c("user", "project"), git = FALSE) {
   scope <- match.arg(scope)
   message("Editing in ", scope, " scope")
 
-  switch(scope, user = path.expand("~"), project = proj_get())
+  if (git) {
+    switch(scope, user = git_user_dot_home(), project = proj_get())
+  } else {
+    switch(scope, user = path.expand("~"), project = proj_get())
+  }
+}
+
+git_user_dot_home <- function() {
+  if (.Platform$OS.type == "windows") {
+    Sys.getenv("USERPROFILE")
```

Comment:
Why I think this is good enough for now: https://github.com/r-lib/usethis/issues/183

Reply:
I think it would be a bit cleaner to separate the git stuff out of scope_dir into a separate function, since the logic for git is obviously much more complicated.